safenode/src/bin/kadnode.rs

```diff
@@ -28,12 +27,33 @@ async fn main() -> Result<()> {
     let _log_appender_guard = init_node_logging(&opt.log_dir)?;
 
     let socket_addr = SocketAddr::new(opt.ip, opt.port);
+    let peers = parse_peer_multiaddreses(&opt.peers)?;
 
     info!("Starting a node...");
-    let (_node, node_events_channel) = Node::run(socket_addr).await?;
+    let (node, node_events_channel) = Node::run(socket_addr).await?;
```

Comment:
this conflicts with the current code base that having the returned `node` removed totally?